<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Registro Empleados</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnListar">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>80</y>
      <width>107</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>17</x>
      <y>40</y>
      <width>671</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableView">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>151</y>
      <width>531</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>127</width>
      <height>341</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dni:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDni"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Nombres:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombres"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apellido Paterno:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtApellidoPaterno"/>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Apellildo Materno:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtApellidoMaterno"/>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Dirección:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtDireccion"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Teléfono:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTelefono"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>541</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnRegistrar">
       <property name="text">
        <string>Registrar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConsultar">
       <property name="text">
        <string>Consultar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEliminar">
       <property name="text">
        <string>Eliminar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnModificar">
       <property name="text">
        <string>Modificar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnQuitar">
       <property name="text">
        <string>Quitar</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>711</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRegistro">
    <property name="title">
     <string>Registro</string>
    </property>
    <addaction name="actionClientes"/>
    <addaction name="actionEmpleados"/>
    <addaction name="actionProductos"/>
    <addaction name="actionProveedor"/>
   </widget>
   <widget class="QMenu" name="menuComprobantes">
    <property name="title">
     <string>Comprobantes</string>
    </property>
    <addaction name="actionVista_Previa"/>
   </widget>
   <addaction name="menuRegistro"/>
   <addaction name="menuComprobantes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClientes">
   <property name="icon">
    <iconset resource="../static/img/proyecto_EDC.qrc">
     <normaloff>:/iconos_main_window/clientes.png</normaloff>:/iconos_main_window/clientes.png</iconset>
   </property>
   <property name="text">
    <string>Clientes</string>
   </property>
  </action>
  <action name="actionEmpleados">
   <property name="icon">
    <iconset resource="../static/img/proyecto_EDC.qrc">
     <normaloff>:/iconos_main_window/empleados.png</normaloff>:/iconos_main_window/empleados.png</iconset>
   </property>
   <property name="text">
    <string>Empleados</string>
   </property>
  </action>
  <action name="actionProductos">
   <property name="icon">
    <iconset resource="../static/img/proyecto_EDC.qrc">
     <normaloff>:/iconos_main_window/venta-de-productos.png</normaloff>:/iconos_main_window/venta-de-productos.png</iconset>
   </property>
   <property name="text">
    <string>Productos</string>
   </property>
  </action>
  <action name="actionProveedor">
   <property name="icon">
    <iconset resource="../static/img/proyecto_EDC.qrc">
     <normaloff>:/iconos_main_window/proveedor.png</normaloff>:/iconos_main_window/proveedor.png</iconset>
   </property>
   <property name="text">
    <string>Proveedor</string>
   </property>
  </action>
  <action name="actionVista_Previa">
   <property name="icon">
    <iconset resource="../static/img/proyecto_EDC.qrc">
     <normaloff>:/iconos_main_window/gui_print_preview_icon_157060.png</normaloff>:/iconos_main_window/gui_print_preview_icon_157060.png</iconset>
   </property>
   <property name="text">
    <string>Vista Previa</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../static/img/proyecto_EDC.qrc"/>
 </resources>
 <connections/>
</ui>
